src/Curiosity/Runtime.hs

```diff
@@ -159,7 +188,8 @@ saveDbAs :: MonadIO m => Runtime -> FilePath -> m (Maybe Errs.RuntimeErr)
 saveDbAs runtime fpath = do
   haskDb <- Data.readFullStmDbInHask $ _rDb runtime
   let bs = Data.serialiseDb haskDb
-  liftIO (try @SomeException (BS.writeFile fpath bs))
+  liftIO
+      (try @SomeException (T.writeFile fpath . TE.decodeUtf8 $ BS.toStrict bs))
```

Comment:
I'm not sure this changes something in practice (other than decoding et reencoding a second time, and forcing a conversion to a strict bytestring. I'm pretty sure that `BS.writeFile` will evaluate completely its argument, write it all to the file, and close the file). Wouldn't serialization to JSON produce a UTF-8 -encoded bytestring anyway ?

The comment at the top of https://hackage.haskell.org/package/aeson-2.1.0.0/docs/Data-Aeson-Text.html says

> Most frequently, you'll probably want to encode straight to UTF-8 (the standard JSON encoding) using encode.

Same comment for the remaining changes below.